frontend/simple/views/CreateGroup.vue

```diff
@@ -1,32 +1,153 @@
+                <p class="select">
+                  <select v-validate data-vv-rules="required" data-vv-as="Conrtibution Privacy" name="contributionPrivacy" v-model="contributionPrivacy">
+                    <option value="">Select an option</option>
+                    <option value="Very Private">Very Private</option>
+                  </select>
+                </p>
+                <span v-if="errors.has('contributionPrivacy')" data-control="contributionPrivacy" class="help is-danger">{{ errors.first('contributionPrivacy') }}</span>
+              </div>
+              <div class="center button-box">
+                <div class="center">
+                  <div id="successMsg" v-if="created" class="created">Success</div>
+                  <button class="button is-success is-large center" type="submit">Create Group</button>
+                </div>
+              </div>
+            </div>
           </div>
-        </div>
-        <div class="column">
-          <h2 class="subtitle">Member your group</h2>
-          <div class="box">
-            <b>What is your group's name?</b>
-          </div>
-        </div>
-        <div class="column">
-          <h2 class="subtitle">Resource allocation</h2>
-          <div class="box">
-            <b>What is your group's name?</b>
-          </div>
-        </div>
+        </form>
       </div>
+      <div class="column is-1"></div>
     </div>
   </section>
 </template>
-
 <script>
+import backend from '../js/backend'
+import {Events} from '../../../shared/events'
+
 export default {
-  name: 'CreateGroupView'
+  name: 'CreateGroupView',
+  methods: {
+    submit: async function () {
+      let success
+      try {
+        success = await this.$validator.validateAll()
+      } catch (ex) {
+        let firsErr = document.querySelector('span.help.is-danger')
+        let control = firsErr.dataset.control
+        let ctrlEl = document.querySelector(`input[name="${control}"], textarea[name="${control}"]`)
+        ctrlEl.focus()
+        ctrlEl.scrollIntoView()
+      }
+      if (success) {
+        let entry = new Events.Group({
+          groupName: this.groupName,
+          sharedValue: this.sharedValue,
+          changePercentage: this.changePercentage,
+          openMembership: this.openMembership,
+          memberApprovalPercentage: this.memberApprovalPercentage,
+          memberRemovalPercentage: this.memberRemovalPercentage,
+          incomeProvided: this.incomeProvided,
+          contributionPrivacy: this.contributionPrivacy,
+          founderHashKey: 'alsdfjlskdfjaslkfjsldkfj'
+        })
+        let hash = entry.toHash()
+        await backend.subscribe(hash)
+        let res = await backend.publishLogEntry(hash, entry, hash)
+        if (!res.ok) {
+          console.error('failed to create group:', res.text)
+        } else {
+          this.created = true
+          console.log('group created. server response:', res.body)
```

Comment:
Since I just sent a PR that changes how some of this stuff is done, and changes the names of some things (like `GroupContract` instead of `Group`), let's get those changes reviewed and merged first, and then merged into your branch, and another commit sent to update this and any other files accordingly.

I recommend deleting your 1-line change in `state.js` before doing that though, as otherwise you'll get a merge conflict in that file.